eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSSplitWriter.java

```diff
@@ -92,8 +92,6 @@ public void write(@Nonnull List<? extends SubmittedVariantOperationEntity> submi
             throws MongoBulkWriteException, AccessionCouldNotBeGeneratedException {
         for (SubmittedVariantOperationEntity entity: submittedVariantOperationEntities) {
             writeRSSplit(entity);
-            this.mongoTemplate.remove(entity,
```

Comment:
The deletion of these candidate entries in the operations table was moved to a separate step after RS Merge and split processing steps because doing these deletes within these steps caused issues with the Merge and split candidate readers.